src/pages/projectCreate/hooks/useWalletForm.tsx

```diff
@@ -310,5 +319,7 @@ export const useWalletForm = ({
     setConnectionOption,
     createWalletInput,
     isLightningAddressInValid,
+    feePercentage,
+    setFeePercentage,
```

Comment:
Maybe use a similar pattern as above, like for node and lightningAddress.

fees: { value, set}

Since you need to pass this around repetitively, it'll be easier.